### PR TITLE
[T2 profile]: Skipping checks for modules part of absent module list (Cisco-8000)

### PR DESCRIPTION
**Pull request by vperumal**
### Description of PR
Even though the script provides an option for skip_module_list. It is used only for presence check. The test currently fails on distributed chassis, wherein it tries to check values for absent modules.

Summary:
Fixes # (issue)

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012

#### What is the motivation for this PR?
Skip modules that are absent in the chassis

#### How did you do it?
Check if the module is part of skip_module list. If yes, skip the test for that module

#### How did you verify/test it?
Verified on Cisco-8000 platform

### CODE REVIEW

**Comment by vperumal** [comment]
FYI @abdosi

File: tests/platform_tests/api/test_module.py [hunk 1]
```diff
@@ -120,6 +120,9 @@ def test_get_model(self, duthosts, enum_rand_one_per_hwsku_hostname, localhost,
             if self.skip_absent_module(i,platform_api_conn):
                 continue
             model = module.get_model(platform_api_conn, i)
+            name = module.get_name(platform_api_conn, i)
```
Comment (gechiang) [inline tests/platform_tests/api/test_module.py:123]: isn't this already done by line 120 already?
 if self.skip_absent_module(i,platform_api_conn):
                continue
is performing exactly what you are trying to add ?  So why that did not skip over the absent modules?
Can you check your change again?

File: tests/platform_tests/api/test_module.py [hunk 2]
```diff
@@ -130,6 +133,9 @@ def test_get_serial(self, duthosts, enum_rand_one_per_hwsku_hostname, localhost,
             if self.skip_absent_module(i,platform_api_conn):
                 continue
             serial = module.get_serial(platform_api_conn, i)
+            name = module.get_name(platform_api_conn, i)
```
Comment (gechiang) [inline tests/platform_tests/api/test_module.py:136]: isn't this already done by line 133 already?
 if self.skip_absent_module(i,platform_api_conn):
                continue
is performing exactly what you are trying to add ?  So why that did not skip over the absent modules?
Can you check your change again?

File: tests/platform_tests/api/test_module.py [hunk 3]
```diff
@@ -140,6 +146,9 @@ def test_get_status(self, duthosts, enum_rand_one_per_hwsku_hostname, localhost,
             if self.skip_absent_module(i,platform_api_conn):
                 continue
             status = module.get_status(platform_api_conn, i)
+            name = module.get_name(platform_api_conn, i)
```
Comment (gechiang) [inline tests/platform_tests/api/test_module.py:149]: isn't this already done by line 146 already?
 if self.skip_absent_module(i,platform_api_conn):
                continue
is performing exactly what you are trying to add ?  So why that did not skip over the absent modules?
Can you check your change again?

**Comment by vperumal** [comment]
Hi @gechiang, when I opened the PR, that check was not there. But now the functionality is present, closing this PR

**Pull request closed without merging**